src/main/java/se/lth/cs/connect/routes/Account.java

```diff
@@ -183,6 +185,11 @@ protected void setup(PippoSettings conf) {
             String email = AccountSystem.verifyEmail(token);
             if (email == null)
                 throw new RequestException("Invalid token: " + token);
+            
+            if(!frontend.equals("http://localhost:8181")){ //Dont wan't to send mail to serpconnect@cs.lth.se when we are testing.
+            	app.getMailClient().
+    				sendEmail("serpconnect@cs.lth.se", "SERP connect email registration", "The following email has registered to SERP-connect:\n\n" + email);
```

Comment:
@Timbozand email addr and body needs to be configurable in the application.properties and messages_en.properties filef